.gitignore

```diff
@@ -42,4 +42,5 @@ testem.log
 .DS_Store
 Thumbs.db
 .npmrc
-/.nx
\ No newline at end of file
+/.nx
+chain-cli/bin/run
```

Comment:
I think we acually need this to make cli work

Reply:
Makes sense, removed that line!